### PR TITLE
Workspace for Espressif's SoC development on Zephyr update

### PR DESCRIPTION
**Pull request by wmrsouza**
Workspace for Espressif's SoC development on Zephyr update:
- toolchain version
- .svd files
- ESP32C2 workspace added

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -82,14 +82,43 @@
...
 			"mon halt",
-			"flushregs"
+			"maintenance flush register-cache"
 		],
 		"overrideRestartCommands": [
 			"mon reset halt",
-			"flushregs",
+			"maintenance flush register-cache",
 		],
 		// "runToEntryPoint": "main"
```
Comment (LucasTambor) [inline .vscode/launch.json]: Can you remove the commented code?

File: .vscode/launch.json [hunk 2]
```diff
@@ -174,13 +203,15 @@
 		"overrideAttachCommands": [
 			"set remote hardware-watchpoint-limit 2",
 			"mon halt",
-			"flushregs",
+			"maintenance flush register-cache",
 		],
 		"overrideRestartCommands": [
 			"mon reset halt",
-			"flushregs",
+			//"thb __start",
```
Comment (LucasTambor) [inline .vscode/launch.json]: ditto

File: .vscode/launch.json [hunk 3]
```diff
@@ -174,13 +203,15 @@
...
+			"maintenance flush register-cache",
 		],
 		"runToEntryPoint": "main",
+		//"showDevDebugOutput": "raw",
```
Comment (LucasTambor) [inline .vscode/launch.json]: ditto

**Review by LucasTambor — CHANGES_REQUESTED** [review]
Niice

**Comment by wmrsouza** [comment]
@LucasTambor

**Review by LucasTambor — APPROVED** [review]

**Pull request merged**